.github/workflows/dispatch-workflow.yml

```
@@ -57,3 +57,12 @@ jobs:
     secrets: inherit
     with:
       profile: ${{ inputs.profile }}
+
+  OpenHarmony:
```

```suggestion
  ohos:
```
to be more consistent with the rest